docs/components/assets.rst

```diff
@@ -137,5 +137,24 @@ It's possible to delete an Asset by clicking on the 'delete' button while viewin
 
 .. warning:: 
     Once deleted, you can't retrieve an Asset, and statistics relating to the number of downloads for that Asset are no longer be available. Contact Points accumulated as a result of accessing the resource remain. It's recommended where possible to unpublish Assets which are no longer in use - in future there may be an archive feature.
-    
 
+Display assets directly in the browser
+--------------------------------------
```

Comment:
```suggestion
============================
```

Reply:
This has to be the same length as the heading - could not easily check this in the GitHub UI!